rigex-front-end/src/store/login/action.js

```diff
@@ -1,10 +1,12 @@
 import actionTypes from './actionTypes';
 import routePaths from '../../constants/routePaths';
 import history from '../../utils/history';
+
 const setLoggedInUser = (user) => {
   return (dispatch) => {
     dispatch({ type: actionTypes.SET_USER, user: { ...user } });
     localStorage.setItem('userId', JSON.stringify(user.id));
+
     history.push(routePaths.landing);
```

Comment:
if you use `history.replace` it will prevent the user from clicking the back button into the login screen